k6/src/modules/auth/login.js

```diff
@@ -0,0 +1,62 @@
+import http from 'k6/http';
+import { check } from 'k6';
+import User from './model/User.js';
+
+/**
+ * @param params
+ * @returns User
+ */
+export function loginUi(params) {
+    const res = http.request(
+        'POST',
+        params.url + '/tao/Main/login',
+        {
+            login: params.login,
+            password: params.password,
+            loginForm_sent: 1,
+            connect: 'Log in'
+        },
+        {
+            redirects: 999,
+            headers: {
+                'Upgrade-Insecure-Requests': 1,
+                Origin: params.url,
+                'Content-Type': 'application/x-www-form-urlencoded',
+                Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9'
+            }
+        }
+    );
+
+    check(res, { 'Login status was 200': r => r.status === 200 });
+
+    return getUser(params);
```

Comment:
Shouldn't we get a cookie after login? Maybe we can get the user already after login, instead of doing an additional call.

Reply:
I will investigate again. Last time I could not get this cookies in the response headers.

Reply:
I made it work by not going to the last redirect 302. So I am asserting just the 302 response.